charts/nfdiv-cms/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
 appVersion: "1.0"
 description: Divorce - Case Maintenance Service
-name: div-cms
+name: nfdiv-cms
```

Comment:
we should also rename this folder charts/div-cms -> charts/nfdiv-cms

Reply:
Unsure what you mean as I've changed the folder name to charts/nfdiv-cms?